clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/domain/LoadBalancerResolver.groovy

```diff
@@ -19,19 +19,81 @@ package com.netflix.spinnaker.clouddriver.openstack.domain
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 
+/**
+ * Load balancer description is a comma-deliminated set
+ * of key-value pairs.
+ *
+ * Usually this will be in the form of "internal_port=8100,created_time=12345678", for example.
+ */
 trait LoadBalancerResolver {
 
-  final String descriptionRegex = ".*internal_port=([0-9]+).*"
-  final Pattern descriptionPattern = Pattern.compile(descriptionRegex)
+  final String portRegex = ".*internal_port=([0-9]+).*"
+  final Pattern portPattern = Pattern.compile(portRegex)
+  final String createdRegex = ".*created_time=([0-9]+).*"
+  final Pattern createdPattern = Pattern.compile(createdRegex)
+
+  /**
+   * Parse the internal port from a load balancer description in the following format.
+   * <br><br>
+   * {@code
+   *  ...,internal_port=8100,...
+   * }
+   * @param description
+   * @return the port value
+   */
+  int parseInternalPort(final String description) {
```

Comment:
This seems like a good time to use `Integer` and return `null` on an invalid input